### DISCUSSION TITLE
Btt pad 7 multiple instances help setup

### DISCUSSION THREAD

**Discussion by PPPRY** [bassamanator/Sovol-SV06-firmware#106] [Q&A]
Klipper reports: ERROR

Include file '/home/biqu/printer_1_data/config/././cfgs/misc-macros.cfg' does not exist

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

**Comment by bassamanator** [comment]
The path to that file is incorrect, make the appropriate adjustment.
